src/views/InvolvementsAll/components/Requests/index.jsx

```diff
@@ -136,8 +136,9 @@ const Requests = ({ profile, session }) => {
   }
 
   return (
-    <Grid item xs={12} lg={8}>
-      <Card className={styles.requests}>{content}</Card>
+    <Grid item xs={12} lg={12}>
+      {/* <Card className={styles.requests}></Card> */}
```

Comment:
Before we merge, you should make sure to remove any code that you commented out.